diff --git a/CMakeLists.txt b/CMakeLists.txt
index e912d5d..044ca1a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -135,12 +135,12 @@ if (NOT WIN32)
         add_subdirectory(bin/kmstool-enclave-cli)
     endif()
     
-    find_package(Doxygen OPTIONAL_COMPONENTS dot mscgen dia)
-
-    if (DOXYGEN_FOUND) 
-        set(DOXYGEN_GENERATE_HTML YES)
-        set(DOXYGEN_GENERATE_MAN YES)
-        set(DOXYGEN_OPTIMIZE_OUTPUT_FOR_C YES)
-        doxygen_add_docs(docs ALL ${AWS_NITRO_ENCLAVES_HEADERS})
-    endif()
-endif()
\ No newline at end of file
+    #find_package(Doxygen OPTIONAL_COMPONENTS dot mscgen dia)
+
+    #if (DOXYGEN_FOUND) 
+    #    set(DOXYGEN_GENERATE_HTML YES)
+    #    set(DOXYGEN_GENERATE_MAN YES)
+    #    set(DOXYGEN_OPTIMIZE_OUTPUT_FOR_C YES)
+    #    doxygen_add_docs(docs ALL ${AWS_NITRO_ENCLAVES_HEADERS})
+    #endif()
+endif()
